### ISSUE TITLE
Feature: Add `abstract` attributes and methods to `perfectionist/sort-classes` 

### ISSUE THREAD

**Issue by hugoAlt** [azat-io/eslint-plugin-perfectionist#190]
### What rule do you want to change?

`sort-classes`

### Describe the problem

I personally enjoy having abstract attributes and methods at the top of my classes. I believe that `unknown` groups matches them, but think it would be interesting to add them separately. Same for methods.

More generally speaking, would it not be better to have a system similar to `eslint/naming-convention` where users can define `selectors` and `modifiers` for each, rather than enumerating the list of possible selectors? Complex elements like `private static properties` can not be individually matched currently. 

### Code example

```typescript
abstract class TestClass {

  // ...
  
  
  // Sort me please
  abstract attribute: string;

  // Me too
  abstract method(): void;
}
```

### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by hugoAlt** [comment]
@azat-io I can start working on a PR for this if this is fine for you.

**Comment by azat-io** [comment]
@hugoAlt Sure. That would be great 👍

**Comment by azat-io** [comment]
Added in baa701d494baa8d01f977a650080fb52c61dbc46.
Released in [v3.2.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v3.2.0).
